packages/apps/human-app/server/src/modules/jobs-discovery/spec/jobs-discovery.controller.spec.ts

```diff
@@ -54,22 +48,9 @@ describe('JobsDiscoveryController', () => {
       .compile();
 
     controller = module.get<JobsDiscoveryController>(JobsDiscoveryController);
-    jobsDiscoveryService =
-      module.get<JobsDiscoveryService>(JobsDiscoveryService);
   });
 
   it('should be defined', () => {
     expect(controller).toBeDefined();
   });
-
-  // describe('processJobsDiscovery', () => {
```

Comment:
Added back in https://github.com/humanprotocol/human-protocol/pull/2588/commits/e80d95e9e977c54e66cc49e0dda1585dd9905ea8

I would suggest us to leave some comments for other engineers right in the code in case we leave such commented code blocks: when checking such code later it would be easier to understand if it's intentionally commented or by accident

Reply:
agree